rust/macros/module.rs

```diff
@@ -300,6 +300,26 @@ impl ModuleInfo {
 
         info
     }
+
+    /// Parse ["First", "Second"] into Some(vec!["First", "Second"])
```

Comment:
```suggestion
    /// Parses `["First", "Second"]` into `Some(vec!["First", "Second"])`.
```

i.e. please use Markdown, a period at the end, and the third person.

In addition, this does not add `Some(...)` -- it is the caller that does it, no?

I would reword it like this instead, if you want to give an example:

```suggestion
    /// Parses a list of authors.
    ///
    /// For instance, passing `["First", "Second"]` would return a `vec!["First", "Second"]`.
```